lib/y2snapper_common.pm

```diff
@@ -11,6 +11,43 @@ use testapi;
 use utils;
 use version_utils;
 
+=head2 y2snapper_select_current_conf
+
+ y2snapper_select_current_conf($ncurses);
+
+Select Current Configuration on Snapshots screen
+C<$ncurses> is used to check if it is ncurses.
+
+=cut
+sub y2snapper_select_current_conf {
+    my ($self, $ncurses) = @_;
+    $ncurses //= 0;
+    if ($ncurses) {
+        send_key 'alt-tab';    # Focus Current Configuration selection box
+        send_key 'down';       # Expand test configuration selection box
+        send_key 'down';       # Select test configuration
+        send_key 'ret';        # Apply selection
+        send_key 'tab';
+    }
+    else {
+        send_key 'shift-tab';    # Focus Current Configuration selection box
+        send_key 'down';         # Select test configuration
+    }
+}
+
+=head2 y2snapper_adding_new_snapper_conf
+
+ y2snapper_adding_new_snapper_conf();
+
+Setup another snapper config for /test (creating previously a subvolume for it)
+It allows to have more control over diffs amongs snapshots.
+
+=cut
+sub y2snapper_adding_new_snapper_conf {
+    assert_script_run("btrfs subvolume create /test");
+    assert_script_run("snapper -c test create-config /test");
```

Comment:
it would be nice to have /test as variable as it is repetive here

Reply:
In Perl the recommendation is always not to declare variables and use other means. In this case it would even add and extra line. It is more clear in that way.